.github/workflows/ci.yml

```diff
@@ -8,17 +8,15 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
```

Comment:
can you make it `ubuntu-latest` please? as a Nix user, I'm also a big fan of specific versions, but we're here because of doing that exactly 😄

Reply:
Done.